src/commands/poll.ts

```diff
@@ -0,0 +1,54 @@
+import type { Command } from "./type";
+import { emojis } from "../services/pollManager";
+import { extractId } from "./helpers/mentions";
+import { isTextChannel } from "./helpers/channels";
+
+const mutExclRegEx = /^(Y|y|N|n)$/;
+
+export const poll: Command = {
+  name: "poll",
+  description: "Create a poll in a specified channel with given choices",
+  definition: "poll :name :duration :mutExcl #channelArg *",
+  help: "use !poll <name> <duration> <mutual exclusivity> <channel> <comma seperated options>",
+  permission: 1,
+  async execute({ message: command, services, args }) {
+    const { mutExcl, duration, message, channelArg, name } = args;
+    if (!mutExclRegEx.test(mutExcl)) {
+      await command.reply("Invalid mutExcl param");
+      return;
+    }
+
+    const mutExclBool = mutExcl === "Y" || mutExcl === "y" ? true : false;
```

Comment:
Could just force `mutExcl` to always be lowercase so, `mutExcl.toLowerCase() === "y"`. Also, no need to ternary operator here, the result of the condition _is_ the boolean.